src/CMakeLists.txt

```diff
@@ -439,6 +439,7 @@ if( MIOPEN_BACKEND MATCHES "OpenCL" OR MIOPEN_BACKEND STREQUAL "HIPOC" OR MIOPEN
         kernels/xform_out.s
         kernels/gcnAsmBNBwdTrainSpatial.s
         kernels/MIOpenTensorKernels.cl
+        kernels/MIOpenTensorKernelsHip.cpp
```

Comment:
Perhaps not add `Hip` to the filename since the same is indicated by the extension

Reply:
I've tried, I've got name collision. Probably it could be better to rename opencl file to something else and keep hip filename more consistent.